### PR TITLE
Articles/issue 121 sp event types

### PR DESCRIPTION
**Pull request by mauroservienti**
Related to #121

Article also needs to be linked to the index

### CODE REVIEW

File: servicepulse/event-types.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+* `Failed`: The message has failed and has arrived for the first time in the error queue;
+* `RepeatedFailure`: The message has failed multiple times;
+* `ArchivedFailure`: The message has been archived;
+
+More details on [Failed Message Monitoring in ServicePulse](http://docs.particular.net/servicepulse/intro-failed-messages).
+
+### Custom Check / Periodic Check
+
+Custom checks allows an endpoint to notify ServicePulse if a condition, that is business related, is not met. The endpoint heartbeat signals that the endpoint is running, a custom check can add more information that are business related, such as the endpoint is running and can access the external resources required to operate correctly.
```
Comment (SimonCropp) [inline servicepulse/event-types.md]: > Custom checks allows an endpoint

That does not read well

Comment (SimonCropp) [inline servicepulse/event-types.md]: > that is business related

this is redundant and unclear. just remove it

File: servicepulse/event-types.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+title: ServicePulse events
+summary: 'Introduction to ServicePulse monitoring events'
+tags: [ServicePulse, Concepts, Monitoring Events]
+---
+
+ServicePulse is the Particular platform monitoring front-end, powered by ServiceControl. ServicePulse gives you an overview of the system health, based on endpoints heartbeats and custom checks, and a detailed view of all the failed messages.
+
+Note: You can consume the same information not only via the ServicePulse web interface but also subscribing to ServicePulse events broadcasted by the ServiceControl endpoint.
+
+### Heartbeats
```
Comment (SimonCropp) [inline servicepulse/event-types.md]: not sure if this http://docs.particular.net/servicepulse/custom-notification-and-alerting-using-servicecontol-events (which @sfarmar did recently) effects this page??

**Comment by SimonCropp** [comment]
@mauroservienti r u still working on this?

**Comment by mauroservienti** [comment]
Should be done. Merge & close?

**Comment by SimonCropp** [comment]
@mauroservienti but there are still several outstanding problem with this pr. see above comments

**Comment by mauroservienti** [comment]
forgot to push, done now.

**Comment by SimonCropp** [comment]
@mauroservienti looks good

**Comment by mauroservienti** [comment]
I'll merge and add a link in the menu. Thanks.

**Pull request merged**